### PR TITLE
Add free ES screencast and HTML resources

### PR DESCRIPTION
**Pull request by JCarlosR**
## What does this PR do?
- Add resources 
- Add new header and subheader

### Description
Adding web development, Laravel and database fundamentals resources.

### Why is this valuable?
Laravel is a popular PHP framework, adding this in addition to Symfony will be helpful for newcomers.

### How do we know it's really free?
All links are public and do not require any authentication step. You can quickly visit them and access the content.

### For book lists, is it a book? For course lists, is it a course? etc.
Yes.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-es.md [hunk 1]
```diff
@@ -291,12 +298,18 @@
...
+* [Prepárate para una entrevista técnica PHP](https://programacionymas.com/blog/php-preguntas-entrevista-trabajo) - Blog de "Programacion y más" (HTML)
 * [Programación web avanzada: Ajax y Google Maps](http://rua.ua.es/dspace/bitstream/10045/13176/9/04-ajaxphp.pdf) - Sergio Luján Mora, Universidad de Colima (PDF)
 * [Silex, el manual oficial](http://librosweb.es/libro/silex) - Igor Wiedler, `trl.:` Javier Eguíluz Pérez (HTML)
 * [Symfony 1.4, la guía definitiva](http://librosweb.es/libro/symfony_1_4) - Fabien Potencier, François Zaninotto, `trl.:` Javier Eguíluz Pérez (HTML) [(PDF)](https://openlibra.com/es/book/download/symfony-la-guia-definitiva)
 * [Symfony 2.4, el libro oficial](http://librosweb.es/libro/symfony_2_4/) - Fabien Potencier, Ryan Weaver, `trl.:` Javier Eguíluz Pérez (HTML) [(PDF)](https://openlibra.com/es/book/download/manual-de-symfony2-ver-2-0-12)
 
 
+#### Laravel
+
+* [Optimizar consultas Eloquent](https://programacionymas.com/blog/consultas-laravel-eficientes-usando-eloquent) - Blog de "Programacion y más" (HTML)
```
Comment (davorpa) [inline books/free-programming-books-es.md:310]: Parece ser un post de blog más que un libro. ¿Puedes confirmarlo?

File: books/free-programming-books-es.md [hunk 2]
```diff
@@ -291,12 +298,18 @@
 * [Manual de estudio introductorio al lenguaje PHP procedural](https://web.archive.org/web/20140209203630/http://www.cursosdeprogramacionadistancia.com/static/pdf/material-sin-personalizar-php.pdf) - Eugenia Bahit (PDF)
 * [PHP y Programación orientada a objetos](https://styde.net/php-y-programacion-orientada-a-objetos/) - Duilio Palacios (HTML)
 * [POO y MVC en PHP](https://bibliotecafacet.com.ar/wp-content/uploads/2014/12/eugeniabahitpooymvcenphp.pdf) - Eugenia Bahit (PDF)
+* [Prepárate para una entrevista técnica PHP](https://programacionymas.com/blog/php-preguntas-entrevista-trabajo) - Blog de "Programacion y más" (HTML)
```
Comment (davorpa) [inline books/free-programming-books-es.md:301]: Blog Post?

File: books/free-programming-books-es.md [hunk 3]
```diff
@@ -95,6 +97,11 @@
 * [Teoría de la Computación: Lenguajes, Autómatas, Gramáticas](http://ciencias.bogota.unal.edu.co/fileadmin/Facultad_de_Ciencias/Publicaciones/Archivos_Libros/Libros_Matematicas/_Teoria_de_la_Computacion___lenguajes__automatas__gramaticas._/teoriacomputacion.pdf) - Rodrigo De Castro Korgi (PDF)
 
 
+#### Desarrollo Web
+
+* [Diferencia entre Frontend y Backend](https://programacionymas.com/frontend-vs-backend) - Blog de "Programación y más" (HTML)
```
Comment (davorpa) [inline books/free-programming-books-es.md:102]: Otro Blog Post?

File: casts/free-podcasts-screencasts-es.md [hunk 4]
```diff
@@ -9,6 +10,11 @@
 * [Variados](#variados)
 
 
+### Bases de Datos
+
+* [Conceptos fundamentales: Bases de datos relacionales](https://programacionymas.com/series/aprende-bases-de-datos) - Juan Ramos (screencast)
```
Comment (davorpa) [inline casts/free-podcasts-screencasts-es.md:15]: Sería más razonable tener una playlist de YouTube?: https://www.youtube.com/playlist?list=PLzSFZWTjelbJ01UciHPAWTqUFWesoGr9A

File: books/free-programming-books-es.md (hunk 1 above)
Comment (JCarlosR) [reply books/free-programming-books-es.md:310]: Yes. I understand it's short. I can remove it.

File: books/free-programming-books-es.md (hunk 2 above)
Comment (JCarlosR) [reply books/free-programming-books-es.md:301]: I believe this questions and answers compilation counts as a book. I could share a PDF version of this, but there are other resources shared as HTML too.
Please let me know.

File: books/free-programming-books-es.md (hunk 3 above)
Comment (JCarlosR) [reply books/free-programming-books-es.md:102]: Not part of the blog. Shared as a guide.
We have similar guides in the list.
But again, it's up to you. Just let me know please.

File: casts/free-podcasts-screencasts-es.md (hunk 4 above)
Comment (JCarlosR) [reply casts/free-podcasts-screencasts-es.md:15]: This way they don't get distracted by other videos or ads.
But we can replace with the playlist if you prefer. Just confirm please.

File: books/free-programming-books-es.md (hunk 2 above)
Comment (davorpa) [reply books/free-programming-books-es.md:301]: Please, should be removed

File: books/free-programming-books-es.md (hunk 3 above)
Comment (davorpa) [reply books/free-programming-books-es.md:102]: Please, should be removed

File: books/free-programming-books-es.md (hunk 1 above)
Comment (davorpa) [reply books/free-programming-books-es.md:310]: Yes, please

File: casts/free-podcasts-screencasts-es.md (hunk 4 above)
Comment (davorpa) [reply casts/free-podcasts-screencasts-es.md:15]: Thanks for answering. Replace it, please

Comment (JCarlosR) [reply casts/free-podcasts-screencasts-es.md]: Okay. I did rebase to prevent adding many commits.

File: books/free-programming-books-es.md [hunk 5]
```diff
@@ -5,6 +5,7 @@
     * [Algoritmos y Estructuras de Datos](#algoritmos-y-estructuras-de-datos)
     * [Base de Datos](#base-de-datos)
     * [Ciencia Computacional](#ciencia-computacional)
+    * [Desarrollo Web](#desarrollo-web)
```
Comment (davorpa) [inline books/free-programming-books-es.md:8]: Remove missing section in TOC

```suggestion
    * [Desarrollo Web](#desarrollo-web)
```

**Review by davorpa — APPROVED** [review]

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**